### DISCUSSION TITLE
Spotify playing while account is being used elsewhere

### DISCUSSION THREAD

**Discussion by seschu** [MiczFlor/RPi-Jukebox-RFID#2442] [General]
Hi there,

My question is: Why does this work? I am confused. With my other devices with the official Spotify app only one of them can stream music if they are all connected to the Internet. The phoniebox is also "online" but still it can stream music in parallel . So how come?

**Comment by Micharius** [comment]
True, I noticed that too. For me it's an advantage, my daughter can listen to her stories while I'm listening to my favourite podcast. Must be at technical issue at Spotify I guess. Don't tell anybody 😀
